_data/errors.yml

```diff
@@ -195,3 +195,31 @@ oidc:
         There is no registered certificate that matches the signature of the `client_assertion` JWT that is being passed as part of the token. request.
         ##### What to do:
         - Ensure that the public certificate that matches the private key used to sign the JWT is registered in your application's configuration in the [Partner Dashboard](https://dashboard.int.identitysandbox.gov/){:target="_blank"}.
+  userinfo:
+    - title: Malformed Authorization header
+      id: malformed-auth-header
+      accordion-id: malformed_auth_header
+      content: |
+        ##### Why it's happening
+        The `userinfo` endpoint requires an Authorization Header that includes a Bearer token. The Bearer token value is the `access_token` value from the `token` endpoint response.
+        ##### What to do:
+        - Ensure you are sending an [Authorization header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization){:target="_blank"} in your request to the `userinfo` endpoint.
```

Comment:
Can you also add `{:class="usa-link--external"}` after the url to indicate that it is an external url? Also for all of the other mozilla urls like on line 215 and 224

Reply:
added in https://github.com/18F/identity-dev-docs/pull/398/commits/4bb0d783af1e2c2a834fc8fcaadfe47a09c24370